components/Core/Carousel/index.tsx

```diff
@@ -0,0 +1,49 @@
+import React, { FC, useState } from 'react'
+import { FlatButton, VerticalLayout, HorizontalLayout } from '../'
+import styles from './Carousel.module.css'
+
+interface CarouselProps {
+  children: React.ReactNode[]
```

Comment:
In React.FC definition, children is already part of its interface and this shouldn't be required